cmd/build.go

```diff
@@ -32,7 +33,7 @@ func Build(ctx context.Context) *cobra.Command {
 
 	options := build.BuildOptions{}
 	cmd := &cobra.Command{
-		Use:   "build [PATH]",
+		Use:   "build [service]",
```

Comment:
while we don't document the new build command and the new okteto manifest, I would keep documenting the current behavior. Same for `-f`